todo/src/reducers/reducers.js

```diff
@@ -0,0 +1,55 @@
+import {
+  ADD_TODO,
+  TOGGLE_COMPLETED,
+  DELETE_TODO,
+  DELETE_COMPLETED
+} from '../actions/actions';
+
+const initialState = {
+  todos: [
+    {
+      id: 'be9a374e-0e79-42d7-ae9d-17ae542205ac',
+      value: 'sand the floor',
+      completed: false
+    },
+    {
+      id: '2a4add54-cbca-40ea-bb07-fa3e0b67618a',
+      value: 'paint the fence',
+      completed: false
+    },
+    {
+      id: 'e12a5b08-9ab3-4aab-805f-905c811e431b',
+      value: 'wax on',
+      completed: false
+    },
+    {
+      id: 'b5d28ffa-8b34-48b4-bbb6-a1085b90d04b',
```

Comment:
interesting -- I think since these are hard-coded within your initial state and these ids could never be replicated with your new todos' uuid-generated ids, you could just use 1-4 here if you wanted